README.md

```diff
@@ -21,7 +22,12 @@ To simplify dependency management, this repository includes several root-level s
 #### For Local Development
 - To start, execute `npm install` from a terminal in the root of this repo.
 - Next, run `npm run load-starter-content` to fetch a starter database and a little bit of starter media to go with it. You will also be prompted to set an admin password.
-- Next, open a terminal instance at the root of each folder (`frontend` and `backend`). Each project needs to be provided with an `APOS_EXTERNAL_FRONT_KEY` environment variable set to the same string value in order to authenticate communication. For example, in each terminal execute `export APOS_EXTERNAL_FRONT_KEY=my-secret-key`.
+- Open terminals:
+ - Mac/Linux users: Open one terminal in `frontend` folder and one in `backend` folder
+ - Windows users:
+   - Open a WSL terminal for the `backend` folder
+   - Open a Windows terminal for `frontend` folder (optional - can also use WSL)
```

Comment:
I take your point but I suggest reversing it because consistency is nicer to most people

Reply:
Done.